src/stories/LinkButton.stories.tsx

```diff
@@ -0,0 +1,49 @@
+import type { Meta, StoryObj } from "@storybook/react";
+import LinkButton from "~/components/LinkButton";
+
+const meta: Meta<typeof LinkButton> = {
+  title: "LinkButton",
+  component: LinkButton,
+  argTypes: {
+    variant: {
+      options: [
+        "primary",
+        "secondary",
+        "tertiary",
+        "ghost-primary",
```

Comment:
`primary`, `secondary`, `tertiary` is not required
go for this and remove everything else:
```
options: [
  "primary",
  "secondary"
]
```